src/resources/swagger.yaml

```diff
@@ -602,6 +602,33 @@ paths:
           description: Active submission is not in PENDING_APPROVAL state and cannot be reopned
         '500':
           $ref: '#/components/responses/ServerError'
+  /submission/program/{programId}/clinical/committed/tsv:
```

Comment:
this is not a submission endpoint, it's in the clinical data domain
`/clinical/program/{programId}/tsv-export`